tests/models/test_python_api.py

```diff
@@ -123,14 +122,34 @@ def predict(self, context, model_input):
         requirements_override.append("conda-forge::charset-normalizer")
 
     mlflow.models.predict(
-        model_uri=f"runs:/{run_id}/model",
+        model_uri=model_info.model_uri,
         input_data={"inputs": [1, 2, 3]},
         content_type=_CONTENT_TYPE_JSON,
         pip_requirements_override=requirements_override,
         env_manager=env_manager,
     )
 
 
+def test_predict_with_extra_envs():
```

Comment:
Can we add a negative test here to ensure that not setting the envvar will raise a specific exception? 
Also, we should add a test once we have error handling for non-virtual env to validate that we throw the right exception for if this arg is set with those other backend environments

Reply:
Actually I've fixed the API to fetch databricks-auth env vars and serverless env vars in my last PR, and it should copy all env vars existing in os.environ, so I'm not sure if this parameter is super useful now.
Because if you use a model that requires some environment variables in local, they must exist in os.environ. The only broken thing might be the databricks-auth and dbconnect ones.
But I can add a test :)